spec/rubocop/cop/rspec/context_wording_spec.rb

```diff
@@ -228,4 +228,36 @@
       RUBY
     end
   end
+
+  context 'when `Prefixes: [on]' do
+    let(:cop_config) do
+      YAML.safe_load(<<-CONFIG)
+        Prefixes:
+          - on
```

Comment:
Perhaps, to help readers:
```suggestion
          - on # evaluates to true
```